### DISCUSSION TITLE
使用cloudflare无法显示登录框

### DISCUSSION THREAD

**Discussion by xyyy-cn** [bs-community/blessing-skin-server#348] [Q&A]
### Blessing Skin 版本

6.0.0 beta4

### PHP 版本

7.4

### 出现问题时所使用的浏览器

Firefox, Chrome, Safari, Microsoft Edge

### 您正在使用的 Web Server

Nginx

### 您正在使用宝塔吗？

- [ ] 是

### 出现了什么问题？

使用cloudflare cdn后 访问https://skinpages.mcserverx.top:2096/auth/login （cloudflare代理）时候不会显示登录框 而使用源服务器ip却正常显示

### 错误日志

```text
无
```

### 重现步骤

访问https://skinpages.mcserverx.top:2096/auth/login （cloudflare代理）时候不会显示登录框

**Comment by mochaaP** [comment]
@ahmr-bot 你的 baseURL 配置有误，请检查。

**Comment by xyyy-cn** [comment]
啥东西 绑定域名吗

**Comment by wangwang-code** [comment]
cloudflare通常只会代理源服务器的80（http）、8080（http）、443（https）、8443（https）端口的流量，2096端口自然不会代理，也就无法访问
